src/modules/users/users.service.ts

```
@@ -184,25 +184,43 @@ export class UserService {
+        doc_data: createUserDocDto.doc_data as string,
+      });
 
-    return await this.userDocsRepository.save(newUserDoc);
+      const savedUserDoc = await this.userDocsRepository.save(newUserDoc);
+      return new SuccessResponse({
+        statusCode: HttpStatus.OK,
+        message: 'User docs added to DB successfully.',
+        data: savedUserDoc,
+      });
+    } catch (error) {
+      if ((error.code = '23505')) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)